### PR TITLE
Allow 0.7.0 of phpcodesniffer isntaller

### PR DESCRIPTION
**Pull request by TimothyBJacobs**
https://wordpress.slack.com/archives/C02RQBWTW/p1603566293205600?thread_ts=1603552988.204000&cid=C02RQBWTW

https://core.trac.wordpress.org/ticket/51624

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
 		"php": ">=5.6"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "~0.6.0",
+		"dealerdirect/phpcodesniffer-composer-installer": "^0.62 || 0.7.0",
```
Comment (jrfnl) [inline composer.json:16]:
```suggestion
		"dealerdirect/phpcodesniffer-composer-installer": "^0.6.2 || ^0.7.0",
```

**Review by jrfnl — COMMENTED** [review]
Aside from a minor syntax mishap in the `composer.json` file, this change looks good.

File: composer.json (hunk 1 above)
Comment (TimothyBJacobs) [reply composer.json]: 🤦, thanks!

Comment (jrfnl) [reply composer.json]: yvw

**Comment by TimothyBJacobs** [comment]
Fixed in 9eed4bc352c944ce75c3ff1615bc63b7162af88b.

**Pull request closed without merging**